src/templates/article-latest.js

```diff
@@ -0,0 +1,565 @@
+        margin-top: 40px;
+        font-weight: bold;
+
+        & + p {
+            margin-top: 8px;
+        }
+    }
+    & h4 {
+        font-size: 24px;
+        line-height: 36px;
+        margin-top: 40px;
+        font-weight: bold;
+
+        & + p {
+            margin-top: 8px;
+        }
+    }
+    & h5 {
+        font-size: 20px;
+        line-height: 30px;
+        margin-top: 24px;
+        font-weight: bold;
+    }
+    & h6 {
+        font-size: 16px;
+        line-height: 24px;
+        margin-top: 40px;
+        font-weight: bold;
+    }
+    @media ${device.laptop} {
+        max-width: none;
+
+        & p {
+            font-size: 14px;
+        }
+        & ul {
+            > li {
+                font-size: 14px;
+                line-height: 20px;
+            }
+        }
+        & h1 {
+            font-size: 32px;
+            line-height: 40px;
+        }
+        & h2 {
+            font-size: 28px;
+            line-height: 34px;
+        }
+        & h3 {
+            font-size: 24px;
+            line-height: 30px;
+
+            & + p {
+                margin-top: 8px;
+            }
+        }
+        & h4 {
+            font-size: 18px;
+            line-height: 26px;
+
+            & + p {
+                margin-top: 8px;
+            }
+        }
+        & h5 {
+            font-size: 16px;
+            line-height: 24px;
+        }
+        & h6 {
+            font-size: 14px;
+            line-height: 20px;
+        }
+    }
+`
+
+const convertDate = (date) => {
+    const newdate = new Date(date)
+    return (
+        newdate.toLocaleString('en', { day: 'numeric' }) +
+        ' ' +
+        newdate.toLocaleString('en', { month: 'long' }) +
+        ' ' +
+        newdate.toLocaleString('en', { year: 'numeric' })
+    )
+}
+
+const ArticlesTemplate = (props) => {
+    const post_data = props.data.directus.blog[0]
+
+    const footer_banner_data = post_data.footer_banners
+    const side_banner_data = post_data.side_banners
+
+    const side_banner_data_details = {
+        max_w_value: '328px',
+        max_w_tablet: '320px',
+        isExternal: true,
+        redirectLink: side_banner_data.cta_url,
+        imgSrcDesktop: getImage(side_banner_data.banner_image.imageFile),
+    }
+
+    const footer_banner_details = {
+        max_w_value: '792px',
+        max_w_tablet: '580px',
+        isExternal: true,
+        redirectLink: footer_banner_data?.cta_url,
+        imgSrcDesktop: getImage(footer_banner_data?.desktop_banner_image?.imageFile),
+        imgSrcMobile: getImage(footer_banner_data?.mobile_banner_image?.imageFile),
+    }
+
+    return (
+        <Layout>
+            <SEO
+                description={localize(
+                    'Learn about the markets that you can trade online with Deriv, including forex, commodities, synthetic indices, and stock indices.',
+                )}
+                title={localize('Markets | Markets to trade | Deriv')}
+            />
```

Comment:
This will pull from meta title and meta description. We won't need to localize this.

Reply:
Amended in latest commit